docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,132 @@
+3. **Integration with Data Tools**: We believe that being strict about the record format Conduit consumes and produces will make it easier to integrate with other data processing tools.
+
+## Fields
+
+* `.Position` uniquely represents the position of record. This is used to track the position of a record in a source connector, enabling Conduit to resume a stopped pipeline.
+* `.Operation` defines what triggered the creation of a record. There are four possibilities: `create`, `update`, `delete` or `snapshot`. The first three operations are encountered during normal CDC operation, while `snapshot` is meant to represent records during an initial load. Depending on the operation, the record will contain either the payload before the change, after the change, both or none (see fields `.Payload.Before` and `.Payload.After`). 
+* `.Key` represents a value that should identify the entity (e.g. database row).
+* `.Metadata` contains additional information regarding the record.
+* `.Payload.Before` holds the payload before the operation ocurred. These could be present in operations such as `update` and `delete`.
+* `.Payload.After` holds the payload after the operation ocurred. These could be present in operations such as `create`, `snapshot` or `update`.
+
+## Representation
+
+Conduit relies on [protobuf](https://protobuf.dev/) when it comes to defining an OpenCDC record to benefit from the several advantages that it provides. Its definition can be found in the [Buf Schema Registry](https://buf.build/conduitio/conduit-commons/docs/main:opencdc.v1).
+
+When processing records in Conduit, you can always expect this format:
+
+```json
+{
+    "position": "c3RhbmRpbmc=",
+    "operation": "update",
+    "metadata": "{
+        "file.path": "./example.in",
+        "opencdc.readAt": "1663858188836816000",
+        "opencdc.version": "v1"
+    }",
+    "key": "cGFkbG9jay1rZXk=",
+    "payload": {
+        "before": "eWVsbG93",
+        "after": {
+            "bool": true,
+            "float32": 1.2,
+            "float64": 1.2,
+            "int": 1,
+            "int32": 1,
+            "int64": 1,
+            "string": "orange"
+        }
+    }
+}
+```
+
+:::note
+`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded in the example above because these will be a byte slice when represented as JSON.
+:::
+
+## Metadata constants
+
+As part of an OpenCDC record, there will be a set of constants provided for some metadata fields that will vary depending on the connector. These constants can be common to all **OpenCDC** records as part of our standard, some related to our own **Conduit** plugins, and others that will be provided by the **Connector SDK** independently. These constants can be useful to define conventions that will be then used by Conduit to expand its functionality. Here are the ones you can find:
```

Comment:
> some related to our own **Conduit** plugins

Well, not only to plugins, I'd say they are related to Conduit itself, like in general. Like the DLQ metadata fields I mentioned in the other comment, those are not related to a Conduit plugin.

> others that will be provided by the **Connector SDK** independently

The last group are metadata fields provided by connector implementations. Each connector is allowed to add any metadata it wants, the SDK doesn't have anything to do with it.